app.js

```diff
@@ -7,10 +7,9 @@ let bodyparser = require('body-parser');
 let db = pg({host: info.db_host, port: info.db_port, database: info.db_name, user: info.db_user, password: info.db_pass});
 let port = info.server_port;
 let dir = info.parent_dir;
-let image_dir = info.data_dir;
-let images = require('./server/image_response.js')(db, image_dir);
+let images = require('./server/image_response.js')(db);
```

Comment:
You've removed `image_dir` here when it's still referenced and necessary in `image_response.js`